### DISCUSSION TITLE
如何指令发送图片？

### DISCUSSION THREAD

**Discussion by fr1g** [DreamVoid/MiraiMC#131] [问答 / Q&A] [answered]
如果可以的话 我希望能用指令发送/plugins/miraimc/下的图片，用一定的placeholder发送“@全体人员”之类的消息。
或许占位符可以设计为[@all] [pic./img/xxx.png]这样子的？

**Comment by DreamVoid** [comment]
目前没有为指令设计发送富文本消息的功能，即将发布的1.6.1会添加mirai格式码，也会提供本地图片的发送方法

**Comment by DreamVoid** [comment]
@ 全体成员 已在最新开发版解决，参考[Mirai码](https://docs.mirai.mamoe.net/Messages.html#%E7%94%B1-codablemessage-%E5%8F%96%E5%BE%97-mirai-%E7%A0%81%E5%AD%97%E7%AC%A6%E4%B8%B2)发送消息
发送图片仍在开发中，最迟在1.7版本推出。

**Comment by DreamVoid** [comment, marked as answer]
已添加相关方法
